sites/fast-frame/webpack.config.js

```diff
@@ -0,0 +1,41 @@
+const HtmlWebpackPlugin = require("html-webpack-plugin"); // Require  html-webpack-plugin plugin
+
+module.exports = {
+    entry: __dirname + "/src/app/index.js", // webpack entry point. Module to start building dependency graph
```

Comment:
There can be environment issues manually creating paths like this. Check out how the `fast-color-explorer` uses `path.resolve` - this should use a similar pattern